addon/utils/is-fastboot.js

```diff
@@ -0,0 +1,10 @@
+import { computed } from '@ember/object';
+import { getOwner } from '@ember/application';
+
+export default function isFastBoot() {
+  return computed(function() {
+    const fastboot = getOwner(this).lookup('service:fastboot');
+
+    return fastboot ? fastboot.get('isFastBoot') : false;
```

Comment:
I think this should fail if the lookup for `service:fastboot` lookup fails and not silently fall back to returning `false`.

Reply:
👍